--- a/extension/qt/6.8.3/qt_libraries.bzl	2026-04-30 12:10:48.817872690 +0900
+++ b/extension/qt/6.8.3/qt_libraries.bzl	2026-04-30 12:10:48.822964641 +0900
@@ -43,16 +43,16 @@
     # ("ui_tools", "QtUiTools", "Qt6UiTools", [":qt_opengl_widgets", ":qt_widgets", ":qt_gui", ":qt_core"]),
     ("virtual_kerboard", "QtVirtualKeyboard", "Qt6VirtualKeyboard", [":qt_quick", ":qt_qml", ":qt_gui", ":qt_core"]),
     ("web_channel", "QtWebChannel", "Qt6WebChannel", [":qt_qml", ":qt_core"]),
-    # ("web_engine_core", "QtWebEngineCore", "Qt6WebEngineCore", [
-    #     ":qt_quick",
-    #     ":qt_gui",
-    #     ":qt_web_channel",
-    #     ":qt_qml",
-    #     ":qt_network",
-    #     ":qt_positioning",
-    #     ":qt_core",
-    # ]),
-    # ("web_engine_widgets", "QtWebEngineWidgets", "Qt6WebEngineWidgets", [":qt_quick_widgets", ":qt_web_engine_core", ":qt_print_support", ":qt_widgets", ":qt_gui", ":qt_core"]),
+    ("web_engine_core", "QtWebEngineCore", "Qt6WebEngineCore", [
+        ":qt_quick",
+        ":qt_gui",
+        ":qt_web_channel",
+        ":qt_qml",
+        ":qt_network",
+        ":qt_positioning",
+        ":qt_core",
+    ]),
+    ("web_engine_widgets", "QtWebEngineWidgets", "Qt6WebEngineWidgets", [":qt_quick_widgets", ":qt_web_engine_core", ":qt_print_support", ":qt_widgets", ":qt_gui", ":qt_core"]),
     ("web_sockets", "QtWebSockets", "Qt6WebSockets", [":qt_network", ":qt_core"]),
     ("web_view", "QtWebView", "Qt6WebView", [":qt_core"]),
     ("xml", "QtXml", "Qt6Xml", [":qt_core"]),
